plugins/modules/k8s_exec.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020, Red Hat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: k8s_exec
+short_description: Execute command in Pod
+version_added: "2.10"
```

Comment:
Just wondering how this will work with the new world order, since this is a collection and it's technically not going into Ansible 2.10... I'm going to ask the Ansible team about this field.

Reply:
Let's make this "1.0.0" instead of "2.10" — seems like there _could_ be issues generating HTML docs for this, but worst case I think it will just hide the version_added field contents. And that's a bug we can work on getting fixed.

Reply:
Done